src/components/ha-sidebar.ts

```diff
@@ -310,6 +314,8 @@ class HaSidebar extends LitElement {
         selectedEl.scrollIntoViewIfNeeded();
       }
     }
+
+    this._rtl = computeRTL(this.hass);
```

Comment:
You know update this property only if `SUPPORT_SCROLL_IF_NEEDED`. Move this above the if-check but still only re-calculate it if `hass` has changed.

Reply:
Crap. Fixed.